src/index.js

```diff
@@ -14,11 +16,15 @@ app.use(cors());
 // api doc
 app.use('/api-docs', swaggerUI.serve, swaggerUI.setup(doc));
 
+app.use('/api/v1', routesV1);
+
 app.get('/', (req, res) => res.status(200).send({ message: 'Welcome to Barefoot Nomad' }));
 app.all('*', (req, res) => res.send({ message: 'route not found' }));
 
-app.listen(port, () => {
+models.sequelize.sync().then(() => {
```

Comment:
Why are you calling sequelize in the index.js file? anything sequelize should be done in the controller's or model's folder

Reply:
I've resolved it. Thanks.